DSCResources/MSFT_xDisk/MSFT_xDisk.psm1

```diff
@@ -143,9 +143,20 @@ function Set-TargetResource
         }
         else 
         {
-            Write-Verbose -Message "The volume already exists, adjusting drive letter..."
-            $VolumeDriveLetter = ($Disk | Get-Partition | Get-Volume).driveletter
-            Set-Partition -DriveLetter $VolumeDriveLetter -NewDriveLetter $DriveLetter
+            $Volume = $Disk | Get-Partition | Get-Volume
```

Comment:
Please add or update  test coverage for the scenario you added code for.

It is required that you provide adequate test coverage for the code you change. [Contribution Guidelines -Adding test coverage for DSC resources.](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#adding-test-coverage-for-dsc-resources)  Please work with us to come up with a plan to cover this code.